README.md

```diff
@@ -238,7 +238,6 @@ This section is auto generated. Please update `feature-matrix.json` and then run
 | -------------------------- | :------: |
 | grid                       |          |
 | pages                      |          |
-| print                      |          |
```

Comment:
@lognaturel is it okay to remove this from this section and from the feature-matrix.json? Or would it be better to leave it but add 'no implementing` ?

Reply:
Removing it makes sense to me. We currently don't consider it a requirement. If we do in the future, we can add it back in. I think of this matrix as tracking functionality we know we must have.